README.md

```diff
@@ -1,4 +1,5 @@
 # RimDev ES6 component generator
+![build status](https://ci.appveyor.com/api/projects/status/ae1taoboaxyvy3ou?svg=true)
```

Comment:
This badge won't match the expected GitHub repository. If we want to keep this in using https://shields.io/ may be easier here. However, it will require the username or organization that the repository will live in. Not sure if that's an issue or not.